src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -239,166 +239,168 @@ protected void install(InstallConfig installConfig) {
         }
 
         inboundRateLimiter.clear();
+        config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((what, newv) -> {
+            if (what.equals(WhatHappened.timestampUpdated)) {
+                return;
+            }
+            if (installConfig.configureSynchronizeConfig
+                    && (newv == null || newv.childOf(CONFIGURATION_SYNCHRONIZATION_TOPIC))) {
+                configureSynchronization();
+            }
+            if (installConfig.configureRateLimitsConfig
+                    && (newv == null || newv.childOf(CONFIGURATION_RATE_LIMITS_TOPIC))) {
+                rateLimits();
+            }
+            if (installConfig.configureMaxDocSizeLimitConfig
+                    && (newv == null || newv.childOf(CONFIGURATION_MAX_DOC_SIZE_LIMIT_B_TOPIC))) {
+                maxSizeDocLimitConfig();
+            }
+            if (installConfig.configureSyncDirectionConfig
+                    && (newv == null || newv.childOf(CONFIGURATION_SYNC_DIRECTION_TOPIC))) {
+                syncDirection();
+            }
+            if (installConfig.configureStrategyConfig
+                    && (newv == null || newv.childOf(CONFIGURATION_STRATEGY_TOPIC))) {
```

Comment:
Might be a dumb question but if the config previously had a value for `CONFIGURATION_STRATEGY_TOPIC` and then ONLY that key gets removed from the config would `newv == null`?. I see below we do a call for `toPojo` but would removing the key would cause it to return null or an empty map?

Reply:
`newv` will be the topic that is removed - `services.aws.greengrass.ShadowManager.configuration.strategy` with its old value.